From oliver@neukum.org Wed Jan  3 06:15:05 2007
From: Oliver Neukum <oliver@neukum.org>
To: Greg KH <gregkh@suse.de>, linux-usb-devel@lists.sourceforge.net
Subject: USB: small update to Documentation/usb/acm.txt
Date: Wed, 3 Jan 2007 15:14:43 +0100
Message-Id: <200701031514.43808.oliver@neukum.org>

this docu update mentions that cell phones also use cdc-acm.

Signed-off-by: Oliver Neukum <oliver@neukum.name>

---
 Documentation/usb/acm.txt |    4 ++++
 1 file changed, 4 insertions(+)

--- gregkh-2.6.orig/Documentation/usb/acm.txt
+++ gregkh-2.6/Documentation/usb/acm.txt
@@ -46,6 +46,10 @@ Abstract Control Model (USB CDC ACM) spe
 
 	3Com USR ISDN Pro TA
 
+  Some cell phones also connect via USB. I know the following phones work:
+
+	SonyEricsson K800i
+
   Unfortunately many modems and most ISDN TAs use proprietary interfaces and
 thus won't work with this drivers. Check for ACM compliance before buying.
 
